### ISSUE TITLE
Security vulnerabilities in 1.21.4-alpine

### ISSUE THREAD

**Issue by KirylBubovich** [nginxinc/docker-nginx#598]
I'm getting the following `busybox` vulnerabilities:
```
+------------+------------------------+--------------+-----------------+---------------------------------------------------------------+
| Unapproved | Unknown CVE-2021-42378 | busybox      | 1.33.1-r3       |                                                               |
|            |                        |              |                 | https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2021-42378 |
+------------+------------------------+--------------+-----------------+---------------------------------------------------------------+
| Unapproved | Unknown CVE-2021-42380 | busybox      | 1.33.1-r3       |                                                               |
|            |                        |              |                 | https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2021-42380 |
+------------+------------------------+--------------+-----------------+---------------------------------------------------------------+
| Unapproved | Unknown CVE-2021-42386 | busybox      | 1.33.1-r3       |                                                               |
|            |                        |              |                 | https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2021-42386 |
+------------+------------------------+--------------+-----------------+---------------------------------------------------------------+
| Unapproved | Unknown CVE-2021-42375 | busybox      | 1.33.1-r3       |                                                               |
|            |                        |              |                 | https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2021-42375 |
+------------+------------------------+--------------+-----------------+---------------------------------------------------------------+
| Unapproved | Unknown CVE-2021-42379 | busybox      | 1.33.1-r3       |                                                               |
|            |                        |              |                 | https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2021-42379 |
+------------+------------------------+--------------+-----------------+---------------------------------------------------------------+
| Unapproved | Unknown CVE-2021-42385 | busybox      | 1.33.1-r3       |                                                               |
|            |                        |              |                 | https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2021-42385 |
+------------+------------------------+--------------+-----------------+---------------------------------------------------------------+
| Unapproved | Unknown CVE-2021-42383 | busybox      | 1.33.1-r3       |                                                               |
|            |                        |              |                 | https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2021-42383 |
+------------+------------------------+--------------+-----------------+---------------------------------------------------------------+
| Unapproved | Unknown CVE-2021-42381 | busybox      | 1.33.1-r3       |                                                               |
|            |                        |              |                 | https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2021-42381 |
+------------+------------------------+--------------+-----------------+---------------------------------------------------------------+
| Unapproved | Unknown CVE-2021-42382 | busybox      | 1.33.1-r3       |                                                               |
|            |                        |              |                 | https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2021-42382 |
+------------+------------------------+--------------+-----------------+---------------------------------------------------------------+
| Unapproved | Unknown CVE-2021-42374 | busybox      | 1.33.1-r3       |                                                               |
|            |                        |              |                 | https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2021-42374 |
+------------+------------------------+--------------+-----------------+---------------------------------------------------------------+
| Unapproved | Unknown CVE-2021-42384 | busybox      | 1.33.1-r3       |                                                               |
|            |                        |              |                 | https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2021-42384 |
+------------+------------------------+--------------+-----------------+---------------------------------------------------------------+
```
This temporary solution is currently being used to avoid vulnerabilities:
`RUN apk add --update --no-cache 'busybox>=1.34.1-r2' --repository='http://dl-cdn.alpinelinux.org/alpine/edge/main'`

**Comment by thresheek** [comment]
This can only be fixed if a fixed busybox version is backported to Alpine 3.14, or when we move to Alpine 3.15 (when it's released).

**Comment by thresheek** [comment]
I think it's now fixed in the current `nginx:1.21.4-alpine`, since it ships busybox 1.33.1-r6, which has some fixes.

Can you please check?

**Comment by KirylBubovich** [comment]
Hey, @thresheek! Indeed, it's now fixed. Thanks! I think the problem can be closed
